node_lambnik/scripts/fetch-data.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# Fetches development data sets from OpenDataPhilly
+if ! [ -f data/STR_Centerline.zip ]; then
+    curl 'https://phila-gisdata.s3.amazonaws.com/ODP/STR_Centerline.zip' -L -o data/STR_Centerline.zip
```

Comment:
I'm getting this when I run `update` a second time:
```
Attempting to add data/STR_Centerline.zip to database                                                                     
Loading STR_Centerline                                                                                                    
Loading data/STR_Centerline.zip                                                                                           
Attempting to extract data/STR_Centerline.zip                                                                             
Archive:  data/STR_Centerline.zip                                                                                         
  inflating: data/STR_Centerline/Street_Centerline.cpg                                                                    
  inflating: data/STR_Centerline/Street_Centerline.dbf                                                                    
  inflating: data/STR_Centerline/Street_Centerline.prj                                                                    
  inflating: data/STR_Centerline/Street_Centerline.sbn                                                                    
  inflating: data/STR_Centerline/Street_Centerline.sbx                                                                    
  inflating: data/STR_Centerline/Street_Centerline.shp                                                                    
  inflating: data/STR_Centerline/Street_Centerline.shp.xml                                                                
  inflating: data/STR_Centerline/Street_Centerline.shx                                                                    
Field length is an FTDouble with width 19 and precision 8                                                                 
Field shape_len is an FTDouble with width 19 and precision 11                                                             
Shapefile type: Arc
Postgis type: MULTILINESTRING[2]
SET
SET
BEGIN
ERROR:  relation "street_centerline" already exists
ERROR:  current transaction is aborted, commands ignored until end of transaction block
ERROR:  current transaction is aborted, commands ignored until end of transaction block
ERROR:  current transaction is aborted, commands ignored until end of transaction block
```
(and so on--many copies of that error)

The check for table existence is based on the filename, so they need to match (which is another thing to note in the usage message about how this actually finds and loads data).

Reply:
I'll go in to the download script and make sure all of the downloaded filenames match the eventual table names, and add some documentation about adding data sets in this way.